File: solution/1_ReverseString/1_ReverseString/ReverseStringTest.cs
using System;
using System.Text;
using System.Collections.Generic;
using System.Linq;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using reversestring;

namespace test
{
    [TestClass]
    public class ReverseStringTest
    {
        [TestMethod]
        public void reverseLegeString() {
		    Assert.AreEqual("", new ReverseString().reverse(""));
	    }
	
	    [TestMethod]
	    public void reverseEenCharacterString() {
		    Assert.AreEqual("a", new ReverseString().reverse("a"));
	    }

        [TestMethod]
	    public void reverseMeerdereCharacterString() {
		    Assert.AreEqual("abcdefg", new ReverseString().reverse("gfedcba"));
	    }
    }
}

File: workspace/1_ReverseString/test/reversestring/ReverseStringTest.java
package reversestring;

import static junit.framework.Assert.assertEquals;

import org.junit.Test;

public class ReverseStringTest {
	
	@Test
	public void reverseLegeString() {
		assertEquals("", new ReverseString().reverse(""));
	}
	
	@Test
	public void reverseEenCharacterString() {
		assertEquals("a", new ReverseString().reverse("a"));
	}
	
	@Test
	public void reverseMeerdereCharacterString() {
		assertEquals("abcdefg", new ReverseString().reverse("gfedcba"));
	}
	

}
